app/helpers/users_helper.rb

```diff
@@ -34,6 +34,13 @@ def user_action_buttons(user, additional_actions = [])
         :data => { :no_turbolink => true })
     end
 
+    if user != User.current
+      additional_actions << display_link_if_authorized(_("Invalidate JWTs"),
+        hash_for_invalidate_jwt_user_path(:id => user.id).merge(:auth_object => user, :permission => "edit_users"),
+        :method => :patch, :id => user.id,
+        :data => { :confirm => _("Invalidate tokens for %s?") % user.name })
```

Comment:
```suggestion
        :data => { :confirm => _("Invalidate JWTs for %s?") % user.name })
```
I strongly suggest using "JWTs" here because "tokens" might get confused with Personal Access Tokens.